### ISSUE TITLE
Rftools Control unable to read from Endertanks

### ISSUE THREAD

**Issue by Darkere** [ShetiPhian/Localizations-and-Issues#15]
Not sure how easy it is to fix this issue as it probably has to do with how Endertanks store their fluids. Might need additional input from @Mcjty
Version 1.11 -1.5.0.1

**Comment by McJty** [comment]
Just implementing the IFluidHandler capability properly should work. Make sure that 'null' is also supported for facing as that is a legal value and should give access to the internal contents

**Comment by ShetiPhian** [comment]
McJty is correct, 

just don't pass a null face to 1.11-1.5.0.1 that version doesn't have the fix.

**Comment by ShetiPhian** [comment]
I came here to let you know the 1.11 build is up, null is safe, and if you needed a hand I'd take a look.

But after looking things over I see there has been some misunderstandings/confusion.
I thought you created a plugin for RFTools called RFToolsControl and needed a hand getting it to read my tanks.
When I skimmed the first response I thought it was explaining to use the fluid capability and pass null as a side, but McJty actually answered as if you where asking how to make your tank work with RFToolsControl.

Now I see you where trying to use our mods together and it wasn't working.
Sorry for not noticing that sooner.
On the plus side the new EnderTanks build is up on Curse, so everything should work correctly.
